manifests/params.pp

```diff
@@ -63,4 +63,15 @@
 
   $qpid_url = "amqp:ssl:${::fqdn}:5671"
   $candlepin_event_queue = 'katello_event_queue'
+
+
+  if (versioncmp($::operatingsystemrelease, '7.0') < 0) {
+    $tomcat_service = 'tomcat6'
+  } else {
+    $tomcat_service = 'tomcat'
+  }
+
+  $services_directory = '/etc/katello/services.d'
```

Comment:
Shouldn't this include a trailing slash since you're simplify prefixing in the code?